pkgs/by-name/mo/moxnotify/package.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  clang,
+  libclang,
+  makeWrapper,
+  lua5_4,
+  dbus,
+  wayland,
+  wayland-protocols,
+  pipewire,
+  vulkan-loader,
+  libxkbcommon,
+  libGL,
+  sqlite,
+  fontconfig,
+  freetype,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "moxnotify";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "mox-desktop";
+    repo = "moxnotify";
+    rev = "6726af08621072e0c95a147cf4ae63ea66c7e857";
+    hash = "sha256-tTgY/813WaW3K8QKbj6qwCVKOAA8zMqy97Q7Z5qA0JM=";
+  };
+
+  cargoHash = "sha256-o2YyPa7bX9585lsicJjhj1xJ1jMdU5mlxbEn/6zSy8U=";
+
+  nativeBuildInputs = [
+    pkg-config
+    clang
+    makeWrapper
+  ];
+
+  buildInputs = [
+    lua5_4
+    dbus
+    wayland
+    wayland-protocols
+    pipewire
+    vulkan-loader
+    libxkbcommon
+    libGL
+    sqlite
+    fontconfig
+    freetype
+    libclang.lib
+  ];
+
+  # Set LIBCLANG_PATH for bindgen
+  LIBCLANG_PATH = "${libclang.lib}/lib";
```

Comment:
Maybe better to use `env.LIBCLANG_PATH` ?

Reply:
done